test/integration/dss/test_dss_api.py

```diff
@@ -418,5 +419,29 @@ def test_python_login_logout_user_account(self):
         self.assertNotIn("oauth2_token", config)
 
 
+class TestDssApiFileLimit(unittest.TestCase):
+    staging_bucket = "org-humancellatlas-dss-cli-test"
+    old_open_file_limits = resource.getrlimit(resource.RLIMIT_NOFILE)
+
+    def test_python_open_file_limit_upload(self):
+        old_open_file_limits = resource.getrlimit(resource.RLIMIT_NOFILE)
+        bundle_path = os.path.join(TEST_DIR, "upload", "many-files")
+        uploaded_paths = [x.path for x in iter_paths(str(bundle_path))]
+        uploaded_files = [object_name_builder(p, bundle_path) for p in uploaded_paths]
+        client = hca.dss.DSSClient(swagger_url="https://dss.data.humancellatlas.org/v1/swagger.json")
```

Comment:
we should avoid uploading test data into `dss-prod`. please change this to `https://dss.integration.data.humancellatlas.org/v1/swagger.json`
@DailyDreaming

Reply:
`https://dss.data.humancellatlas.org/v1/swagger.json` is the default too so this is a no op.

Reply:
`https://dss.data.humancellatlas.org/v1/swagger.json` is the default too so this is a no op.

Reply:
Let's discuss on Slack. I'll start a thread there.